pkg/entity/batch_exporter/cmd.go

```diff
@@ -39,8 +39,8 @@ func DeleteCmd() *cobra.Command {
 			printer = configurePrinter(cmd)
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			for i, _ := range args {
```

Comment:
This could also be `for _, arg := ...`, and then `del(arg, cmd)` if I'm correct. Now we use the index instead of the value.